src/main/resources/assets/communitymod/models/item/bean_sword.json

```diff
@@ -0,0 +1,6 @@
+{
+  "parent": "minecraft:item/handheld",
```

Comment:
You can remove the `minecraft:`. It's not needed.

Reply:
It's not needed, but they probably did it because Minecraft started doing it too. It's always good to have a note, and since vanilla also does that...

Reply:
nah its completely unnecessary. theres no reason where it would ever be useful to have

Reply:
What's the problem with it, it follows what vanilla does and that's good

Reply:
following vanilla is **not** good. and also to keep consistency, it should be removed

Reply:
y. the mod description literally says it's unorganized. but yeah remove it then

Reply:
thats not what was meant by unorganized. unorganized was meant to mean the features. the code should still be kept somewhat maintainable and consistent